web/src/app/modules/shared/models/content.ts

```diff
@@ -585,3 +585,16 @@ export interface StepperView extends View {
     steps: StepItem[];
   };
 }
+
+export interface IconView extends View {
+  config: {
+    shape: string;
+    size: string;
+    direction: string;
+    flip: string;
+    solid: boolean;
+    status: string;
```

Comment:
I want to change this field to `color` and then create some static helpers called `SuccessColor` , `WarningColor` , `DangerColor` .. I think limiting this to just for colors will be a problem.

Reply:
by static helper do you mean change the mapper for a switch case? I mean 
rather than this
```
 statusHash = {
    info: 'is-info ',
    success: 'is-success ',
    warning: 'is-warning ',
    danger: 'is-error ',
  };
   mapStatus(status: string) {
    if (!status || status === '') {
      return;
    }
    this.classAttr += this.statusHash[status] || '';
  }
```
use this

```

  static SuccessColor = 'is-success';

  mapColor() {
    switch () {
      case: 'success'
        return this.SuccessColor;
    }
    .
    .
     .
  }

```

Reply:
BTW Clarity 5 does a property call colors
https://clarity.design/foundation/icons/api/
![image](https://user-images.githubusercontent.com/6991432/112207590-51bc5200-8be5-11eb-9d6f-472104275756.png)

Reply:
Done!
![image](https://user-images.githubusercontent.com/6991432/112210708-060ba780-8be9-11eb-9000-051bc7a2f0df.png)

Reply:
If both `color` and `status` are set, ignore `color` and use `status` .. other than that, this is great!

Reply:
ohh I misunderstood that, It should work now
![image](https://user-images.githubusercontent.com/6991432/112242002-0328ab00-8c19-11eb-83d8-64d8b90dd215.png)